src/components/FormAlertWrapper.js

```diff
@@ -59,24 +59,24 @@ const FormAlertWrapper = props => (
                 <View style={[styles.flexRow, styles.ml2, styles.flexWrap, styles.flex1]}>
                     {!_.isEmpty(props.message) && props.isMessageHtml && <RenderHTML html={`<muted-text>${props.message}</muted-text>`} />}
 
-                    {!_.isEmpty(props.message) && !props.isMessageHtml
-                        ? <Text style={styles.mutedTextLabel}>{props.message}</Text>
-                        : (
-                            <>
-                                <Text style={styles.mutedTextLabel}>
-                                    {`${props.translate('common.please')} `}
-                                </Text>
-                                <TextLink
-                                    style={styles.label}
-                                    onPress={props.onFixTheErrorsPressed}
-                                >
-                                    {props.translate('common.fixTheErrors')}
-                                </TextLink>
-                                <Text style={styles.mutedTextLabel}>
-                                    {` ${props.translate('common.inTheFormBeforeContinuing')}.`}
-                                </Text>
-                            </>
-                        )}
+                    {!_.isEmpty(props.message) && !props.isMessageHtml && <Text style={styles.mutedTextLabel}>{props.message}</Text>}
+
+                    {_.isEmpty(props.message) && (
+                        <>
+                            <Text style={styles.mutedTextLabel}>
+                                {`${props.translate('common.please')} `}
+                            </Text>
+                            <TextLink
+                                style={styles.label}
+                                onPress={props.onFixTheErrorsPressed}
+                            >
+                                {props.translate('common.fixTheErrors')}
+                            </TextLink>
+                            <Text style={styles.mutedTextLabel}>
+                                {` ${props.translate('common.inTheFormBeforeContinuing')}.`}
+                            </Text>
+                        </>
+                    )}
```

Comment:
NAB: imo an alternative to making nested conditional renders easy to follow is to move the logic to a function and do early returns:

```js
const renderMessage = () => {
        if (_.isEmpty(props.message)) {
            return (
                <>
                    <Text style={styles.mutedTextLabel}>
                        {`${props.translate('common.please')} `}
                    </Text>
                    <TextLink
                        style={styles.label}
                        onPress={props.onFixTheErrorsPressed}
                    >
                        {props.translate('common.fixTheErrors')}
                    </TextLink>
                    <Text style={styles.mutedTextLabel}>
                        {` ${props.translate('common.inTheFormBeforeContinuing')}.`}
                    </Text>
                </>
            );
        }

        if (props.isMessageHtml) {
            return <RenderHTML html={`<muted-text>${props.message}</muted-text>`} />;
        }

        return <Text style={styles.mutedTextLabel}>{props.message}</Text>;
    };
``` 

And just call `{renderMessage()}`

Reply:
Updated! Let me know what you think